docs/topics/gradle.md

```diff
@@ -877,9 +877,16 @@ Each of the options in the following list overrides the ones that came before it
 * If the Gradle daemon's JVM arguments have the `kotlin.daemon.jvm.options` system property – use it in the `gradle.properties` file:
 
  ```properties
-  org.gradle.jvmargs=-Dkotlin.daemon.jvm.options=-Xmx1500m -Xms=500m
+  org.gradle.jvmargs=-Dkotlin.daemon.jvm.options=-Xmx1500m,Xms=500m
   ```
 
+  > In this case, enumerate arguments according to the following rules:
```

Comment:
Three rules in a row; maybe use a bullet list?

Reply:
Also, I'd suggest writing this simpler and shorter. Smth like:
"When passing the arguments, follow these rules:"